uportal-war/src/main/java/org/jasig/portal/jgroups/protocols/DAO_PING.java

```diff
@@ -70,20 +73,24 @@ public boolean isDynamic() {
         return true;
     }
 
+    @Override
     public void start() throws Exception {
         super.start();
-        if(interval > 0) {
-            writer_future=timer.scheduleWithFixedDelay(new WriterTask(), interval, interval, TimeUnit.MILLISECONDS);
+        if (interval > 0) {
+            // Call getAndSavePhysicalAddress(group_addr) at interval
+            writer_future = timer.scheduleWithFixedDelay(new WriterTask(), interval, interval, TimeUnit.MILLISECONDS);
         }
     }
 
+    @Override
     public void stop() {
         final Future<?> wf = writer_future;
-        if(wf != null) {
+        if (wf != null) {
+            // Shutdown timer
             wf.cancel(false);
-            writer_future=null;
+            writer_future = null;
         }
-        
+        // TODO? Remove local_addr from table
```

Comment:
We should test but I believe that will be taken care of by the coordinator responding to a View change.